shiny_app/functions/functions_server.R

```diff
@@ -17,44 +17,41 @@ vline = function(x, width=3.0, color="black", ...) {
   return(l_shape)
 }
 
-annotation = function(frac, y, note, ax, ay, color){
-  ann = list(yref = "paper",
+annotation <- function(frac, y, note, color){
+  ann <- list(yref = "paper",
```

Comment:
Boooo, I love =

Reply:
I know but just trying to keep styling consistent